packages/shared/src/components/cards/v1/ArticlePostCard.tsx

```diff
@@ -117,12 +114,10 @@ export const ArticlePostCard = forwardRef(function PostCard(
                 >
                   {title}
                 </CardTitle>
-                {tagsOnCard && (
-                  <>
-                    <div className="flex flex-1" />
-                    <PostTags tags={post.tags} />
-                  </>
-                )}
+                <>
+                  <div className="flex flex-1" />
+                  <PostTags tags={post.tags} />
+                </>
```

Comment:
This should also work? We needed the `<></>` because it was inside `()`
```suggestion
                <div className="flex flex-1" />
                <PostTags tags={post.tags} />
```